src/content/developers/docs/data-and-analytics/block-explorers/index.md

```diff
@@ -140,10 +126,8 @@ Tokens are a type of contract so they'll have similar data to a smart contract.
 
 ### Network {#network}
 
-Of course there's some data that speaks to the health of the network. These are quite specific to Ethereum's proof-of-work consensus mechanism. When Ethereum transitions to proof-of-stake some of this data will be redundant
+Of course there's some data that speaks to the health of the network.
```

Comment:
I don't love this but it's better than what we have 😅 feel free to edit/expand/delete

```suggestion
Some block data is concerned about the health of Ethereum more holistically.
```